meta-psa/recipes-security/psa-storage/psa-storage.bb

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+
+SUMMARY = "ARM Platform Security Architecture (PSA) Protected Storage Library"
+DEPENDS = ""
+HOMEPAGE = "https://github.com/ARMmbed/psa-storage"
+DESCRIPTION = "A Linux C library reference implementation of the PSA protected_storage.h API"
```

Comment:
As soon as it is mainly providing a library I think that as a good practice we should have `SECTION = "libs"`.

And ideally, the name of the recipe should be `libpsastorage` and all recipes which depend on it should have a `DEPENDS = "libpsastorage"` and/or `RDEPENDS_${PN} = "libpsastorage"`.

Reply:
> And ideally, the name of the recipe should be libpsastorage

I'm not convinced. That would be standard for debian package names, but it's not standard for OE package names.

Reply:
> I'm not convinced. That would be standard for debian package names, but it's not standard for OE package names.

I'm not sure if there is a standard for oe recipes, but you can find recipes names with and with `lib` prefix. It is just a matter of looking at the recipe name and knows that it is a library.

Reply:
Changed in https://github.com/ARMmbed/meta-mbl/pull/619